modules/apps/analytics/analytics-client-js/src/analytics.js

```diff
@@ -5,7 +5,6 @@ import middlewares from './middlewares/defaults';
 import Client from './Client';
 
 import defaultPlugins from './plugins/defaults';
-import fingerprint from './utils/fingerprint';
```

Comment:
Is it possible to get rid of `fingerprint.js`?

Reply:
I think so, it won't be necessary anymore.